tests/components/squeezebox/test_media_player.py

```diff
@@ -436,6 +440,157 @@ async def test_squeezebox_play(
     configured_player.async_play.assert_called_once()
 
 
+async def test_squeezebox_play_media_with_announce(
+    hass: HomeAssistant, configured_player: MagicMock
+) -> None:
+    """Test play service call with announce."""
+    await hass.services.async_call(
+        MEDIA_PLAYER_DOMAIN,
+        SERVICE_PLAY_MEDIA,
+        {
+            ATTR_ENTITY_ID: "media_player.test_player",
+            ATTR_MEDIA_CONTENT_TYPE: MediaType.MUSIC,
+            ATTR_MEDIA_CONTENT_ID: FAKE_VALID_ITEM_ID,
+            ATTR_MEDIA_ANNOUNCE: True,
+        },
+        blocking=True,
+    )
+    configured_player.async_load_url.assert_called_once_with(
+        FAKE_VALID_ITEM_ID, "announce"
+    )
+
+
+async def test_squeezebox_play_media_with_announce_zero(
+    hass: HomeAssistant, configured_player: MagicMock
+) -> None:
+    """Test play service call with announce volume zero."""
+    await hass.services.async_call(
+        MEDIA_PLAYER_DOMAIN,
+        SERVICE_PLAY_MEDIA,
+        {
+            ATTR_ENTITY_ID: "media_player.test_player",
+            ATTR_MEDIA_CONTENT_TYPE: MediaType.MUSIC,
+            ATTR_MEDIA_CONTENT_ID: FAKE_VALID_ITEM_ID,
+            ATTR_MEDIA_ANNOUNCE: True,
+            ATTR_MEDIA_EXTRA: {ATTR_ANNOUNCE_VOLUME: 0},
+        },
+        blocking=True,
+    )
+    configured_player.async_load_url.assert_called_once_with(
+        FAKE_VALID_ITEM_ID, "announce"
+    )
+
+
+@pytest.mark.parametrize(
+    ("announce_volume"),
```

Comment:
Small change, but when there's a single parameter, you don't have to wrap it in tuple, which keeps it parallel with the list of values:
```suggestion
    "announce_volume",
```

Change it in all occurrences.

Reply:
done